Makefile

```diff
@@ -28,3 +28,10 @@ clean-roles:
 
 ansible-galaxy:
 	ansible-galaxy install -r requirements.yml --force
+
+import-gpg-keys:
+	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --keyserver hkp://pgp.mit.edu --search-keys $(var);)
```

Comment:
It is only group_vars/all/secure which is used twice - the two awk lines are different.